art/detection/subsetscanning/scanner.py

```diff
@@ -0,0 +1,126 @@
+""" Subset scanning based on FGSS """
+
+import numpy as np
+from art.detection.subsetscanning.scoring_functions import ScoringFunctions
+from art.detection.subsetscanning.scanningops import ScanningOps
+
+class Scanner:
+    
+    """ Fast generalized subset scan
+    McFowland, E., Speakman, S., & Neill, D. B. (2013).
+    Fast generalized subset scan for anomalous pattern detection. 
+    The Journal of Machine Learning Research, 14(1), 1533-1561. """
+
+    @staticmethod
+    def fgss_individ_for_nets(pvalues, a_max=0.5, score_function=ScoringFunctions.get_score_bj_fast):
```

Comment:
Could you please extend this function's docstring with a description of its arguments and return similar to the other function docstrings in ART?